pkg/properties/properties.go

```diff
@@ -26,7 +26,23 @@ func fileToMap(filename string) (map[string]string, error) {
 		return nil, err
 	}
 
-	return toMap(parseLines(string(buf)))
+	return SlicesToMap(parseLines(string(buf)))
+}
+
+// toMap converts a list of key=value strings into a map.
+func SlicesToMap(configs []string) (map[string]string, error) {
```

Comment:
@brianstrauch using this method to parse kms properties

Reply:
Looks like there's only one slice being passed to this function. Also, make sure you update the Godoc!
```suggestion
func ConfigSliceToMap(configs []string) (map[string]string, error) {
```

Reply:
updated